influxdb_store.go

```diff
@@ -145,6 +173,85 @@ func (in *InfluxDBStore) Trace(id ID) (*Trace, error) {
 	return trace, nil
 }
 
+func mustJSONFloat64(x interface{}) float64 {
+	n := x.(json.Number)
+	v, err := n.Float64()
+	if err != nil {
+		panic(err)
+	}
+	return v
+}
+
+func mustJSONInt64(x interface{}) int64 {
+	n := x.(json.Number)
+	v, err := n.Int64()
+	if err != nil {
+		panic(err)
+	}
+	return v
+}
+
+// Aggregate implements the Aggregator interface.
+func (in *InfluxDBStore) Aggregate(start, end time.Duration) ([]*AggregatedResult, error) {
+	// Find the mean (average), minimum, maximum, std. deviation, and count of
+	// all spans.
+	q := `SELECT MEAN("duration"),MIN("duration"),MAX("duration"),STDDEV("duration"),COUNT("duration") from spans`
```

Comment:
Great to see InfluxDB aggregation features landing here!, I think this is great starting point :star2: 

Also I'd like to mention that [continuous queries](https://docs.influxdata.com/influxdb/v0.12/query_language/continuous_queries/) let us have interesting room for performance  improvements here, meaning we could be down-sampling the N-slowest spans every certain time, that down-sampled spans would be saved on a new measurement, perhaps called `slowest_spans`; so we would end-up querying spans on the `slowest_spans` measurement, I think we can handle this in a new PR.